src/current/v24.2/changefeed-sinks.md

```diff
@@ -227,12 +229,14 @@ Changefeeds can deliver messages to Amazon MSK clusters ([Amazon Managed Streami
 
 {% include_cached new-in.html version="v24.2" %} Changefeeds can deliver messages to MSK and MSK Serverless clusters using AWS IAM roles.
 
-{% comment %}will change out the links here to CRDB tutorials in a follow-up PR.{% endcomment %}
+For initial setup guides, refer to:
 
-For initial setup guides, refer to the AWS documentation:
+- [MSK clusters](https://aws.amazon.com/blogs/apn/how-to-unlock-real-time-data-streams-with-cockroachdb-and-amazon-msk/)
```

Comment:
Should we specify "MSK Provisioned" here?

Reply:
I understand why, but Amazon call the "provisioned" just Amazon MSK, so I was trying to keep to that where possible.